### PR TITLE
Fix missing translation in pagination gap tags

### PR DESCRIPTION
**Pull request by mgmn**
Fix for #24261, where i18n of paginator's gap is fallbacks or displayed as translation missing error "Truncate".

### CODE REVIEW

File: app/views/kaminari/_gap.html.haml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-#
+  Non-link tag that stands for skipped pages...
+  available local variables
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+%span.page.gap
+  = sanitize t('pagination.truncate')
```
Comment (ClearlyClaire) [inline app/views/kaminari/_gap.html.haml:9]: Why the `sanitize` call?

Comment (mgmn) [reply app/views/kaminari/_gap.html.haml:9]: Ah, but `sanitize` allows a some of HTML tags...
Would it be better to call `sanitize t('pagination.truncate'), tags: [], attributes: []` to disallow them?

Comment (mgmn) [reply app/views/kaminari/_gap.html.haml]: ce59dce

**Review by ClearlyClaire — APPROVED** [review]

File: app/views/kaminari/_gap.html.haml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+-#
+  Non-link tag that stands for skipped pages...
+  available local variables
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+%span.page.gap
+  = sanitize t('pagination.truncate'), tags: [], attributes: []
```
Comment (Gargron) [inline app/views/kaminari/_gap.html.haml:9]:
```suggestion
  != t('pagination.truncate')
```

**Review by Gargron — CHANGES_REQUESTED** [review]

**Review by Gargron — APPROVED** [review]

**Pull request merged**